### PR TITLE
Update descriptions and examples in core methods

### PR DESCRIPTION
**Pull request by budnix**
### Context

Update API descriptions for Core.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/handsontable/issues/5136

### Checklist:

- [x] My code follows the code style of this project,
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -3094,11 +3121,11 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    *
    * @example
    * ```js
-   * // using an array of arrays
+   * // Uing an array of arrays.
```
Comment (wojciechczerniak) [inline src/core.js]: Typo in "u(s)ing"

File: src/core.js [hunk 2]
```diff
@@ -1076,16 +1076,14 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
...
    * @function setDataAtCell
-   * @param {Number|Array} row Visual row index or array of changes in format `[[row, col, value], ...]`.
-   * @param {Number} col Visual column index.
-   * @param {String} value New value.
+   * @param {Number|Array} row Visual row index or array of changes in format `[[row, col, value],...]`.
+   * @param {Number} [column] Visual column index.
+   * @param {String} [value] New value.
    * @param {String} [source] String that identifies how this change will be described in the changes array (useful in onAfterChange or onBeforeChange callback).
    */
   this.setDataAtCell = function(row, col, value, source) {
```
Comment (wojciechczerniak) [inline src/core.js]: You've changed jsdoc from `col` to `column` but function parameters and code is not updated.

File: src/core.js (hunk 1 above)
Comment (budnix) [reply src/core.js]: Fixed, thanks.

File: src/core.js (hunk 2 above)
Comment (budnix) [reply src/core.js]: Fixed.

**Pull request merged**